docs/src/routes/components/sidemenu/next-to-page-collapsible/+page.svelte

```diff
@@ -0,0 +1,204 @@
+          </li>
+          <li>
+            Om het menu standaard gesloten te tonen tot de gebruiker het menu opent. Voeg de class <code
+              >sidemenu-closed</code
+            >
+            toe aan de <code>body</code>.
+          </li>
+        </ol>
+
+        <h3>Sluit- en open-knop als icoonknop instellen zonder tekst.</h3>
+        <p>
+          Voeg altijd de knoptekst in de taal van het document toe voor screenreader gebruikers.
+        </p>
+        <ol>
+          <li>
+            Voeg de knoptekst toe binnen een <code>data-open-label</code>. Bijvoorbeeld:
+            <code>data-open-label="Zijbalknavigatie"</code>.
+          </li>
+          <li>
+            Voeg de knoptekst toe binnen een <code>data-close-label</code>. Bijvoorbeeld:
+            <code>data-open-label="Sluit zijbalknavigatie"</code>.
+          </li>
+          <li>
+            Vul de variabele <code>--sidemenu-open-button-icon</code> in.
+          </li>
+          <li>
+            Vul de variabele <code>--sidemenu-close-button-icon</code> in.
+          </li>
+          <li>
+            Zet de variabele <code>--sidemenu-button-font-size</code> op <code>0</code> om de tekst visueel
+            te verbergen.
+          </li>
+        </ol>
+
+        <h3>Pagina laden met een gesloten zijmenu</h3>
+        <p>
+          Om het menu standaard gesloten te tonen tot de gebruiker het menu opent. Voeg de class <code
+            >sidemenu-closed</code
+          >
+          toe aan de <code>body</code>.
+        </p>
+      </section>
+
+      <section id="examples">
+        <h2>Voorbeelden</h2>
+        <h3 id="sidemenu-collapsible">Inklapbaar zijmenu</h3>
+        <h4>Visueel voorbeeld</h4>
+        <div class="resize">
+          <iframe
+            src="{base}/examples/sidemenu-next-to-page-collapsible"
+            title="Voorbeeld"
+            height="320px"
+          ></iframe>
+        </div>
+
+        <h4>HTML-voorbeeld</h4>
+        <Code
+          language="html"
+          code={`
+<body class="sidemenu">
+  <nav data-open-label="Zijbalknavigatie" data-close-label="Sluit zijbalknavigatie" aria-label="Zijbalknavigatie">
+    <!-- Zijmenu content -->
+  </nav>
+
+  <div class="page-content">
+    <header>
+      <!-- Inhoud van de header. -->
+    </header>
+
+    <main>
+      <!-- Overige pagina content. -->
+    </main>
+
+    <footer>
+      <!-- Inhoud van de footer. -->
+    </footer>
+  </div>
+</body>
+`}
+        />
+
+        <h3 id="sidemenu-expandable">Uitklapbaar zijmenu</h3>
+        <p>Het menu is standaard ingeklapt en kan door de gebruiker uitgeklapt worden.</p>
+
+        <ol>
+          <li>Voeg de class <code>sidemenu-closed</code> toe aan de body.</li>
+        </ol>
+
+        <h4>Visueel voorbeeld</h4>
+        <div class="resize">
+          <iframe
+            src="{base}/examples/sidemenu-next-to-page-expandable"
+            title="Voorbeeld"
+            height="320px"
+          ></iframe>
```

Comment:
Dit voorbeeld lijkt een 'stuk' zijmenu te tonen binnen het iframe:

<img width="951" alt="Scherm­afbeelding 2025-01-15 om 21 11 39" src="https://github.com/user-attachments/assets/0605ef3d-32e6-41a4-91d8-264f65c27b95" />

De oorzaak lijkt een extra `<div> en `</div>` om de menu items heen in je voorbeeld `docs/src/routes/examples/sidemenu-in-page-expandable/+page.svelte`, zie screenshot:
<img width="738" alt="Scherm­afbeelding 2025-01-15 om 21 15 49" src="https://github.com/user-attachments/assets/855a6991-8e0c-4107-9ed6-7e552dd8ea99" />

Reply:
Ah, slodrig.